app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to t("keywords.footer.about"),   about_path %></li>
```

Comment:
Sau dấu phẩy có 1 space thôi

với dùng I18n cho toàn bộ text đi, chỗ có chỗ ko, nó tủi thân lắm á